draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -326,11 +326,10 @@ same" matching algorithm for origins are defined in {{RFC6454}}.
 "Safe" HTTP methods include `GET`, `HEAD`, `OPTIONS`, and `TRACE`, as defined
 in Section 4.2.1 of {{RFC7231}}.
 
-The term "public suffix" is defined in a note in Section 5.3 of {{RFC6265}} as
-"a domain that is controlled by a public registry", and are also known as
-"effective top-level domains" (eTLDs). For example, `site.example`'s public
+A "public suffix" is a domain that is controlled by a public registry, such as
+"com", "co.uk", and "pvt.k12.wy.us". For example, `site.example`'s public
```

Comment:
Would it be worth making a comparison to "delegation-centric zone" from [RFC 8499](https://tools.ietf.org/html/rfc8499#page-26)? It's certainly possible for a public suffix to not be delegation-centric (e.g., `github.io`) and there are some delegation-centric zones that are not public suffixes (most notably the DNS root zone itself), but there is nevertheless substantial overlap in both concept (different authority between parent and child domain) and contents.

Reply:
I'm not sure that bringing another concept in would clarify, especially given the discrepancies you note between "delegation-centric zone" and existing user agent understanding of "public suffix". I think I'd prefer to continue using the term that user agents have generally applied to cookies.